src/Evervault/Evervault.php

```diff
@@ -80,6 +80,18 @@ public function decrypt($data) {
         return $this->httpClient->decrypt($data);
     }
 
+    public function createClientSideDecryptToken($data, $expiry) {
+        if (!$data) {
+            throw new EvervaultError('Please provide a payload');
```

Comment:
Can you improve this error message to make it more user friendly?

```suggestion
            throw new EvervaultError('The `$data` parameter is required and ensures the issued token can only be used to decrypt that specific payload.');
```